src/compare-pdf-to-snapshot.ts

```diff
@@ -45,6 +45,7 @@ const maskImgWithRegions =
 
 export type CompareOptions = CompareImagesOpts & {
   maskRegions: MaskRegions
+  mode: 'create' | 'failFast'
```

Comment:
How about using more descriptive names along the lines of https://github.com/moshensky/pdf-visual-diff/issues/59#issuecomment-2101025992?